src/js/view/view.js

```diff
@@ -381,58 +377,64 @@ define([
-                '.jw-active-option',
-                '.jw-nextup-header'
-            ], 'background', 'none ' + inactiveColor);
-
-            // Apply background color
-            addStyle([
-                // general background color
-                '.jw-background-color'
-            ], 'background', 'none ' + backgroundColor);
-
-            addStyle([
-                // for small player, set the control bar gradient to the config background color
-                '.jw-flag-time-slider-above .jw-background-color.jw-controlbar'
-            ], 'background', 'none ' + backgroundColorGradient, true);
-
-            // remove the config background on time slider
-            addStyle([
-                '.jw-flag-time-slider-above .jw-background-color.jw-slider-time'
-            ], 'background', 'transparent', true);
+            if (activeColor) {
+                // Apply active color
+                addStyle([
+                    // Toggle and menu button active colors
+                    '.jw-button-color.jw-toggle',
+                    '.jw-button-color:hover',
+                    '.jw-button-color.jw-toggle.jw-off:hover',
+                    '.jw-option:not(.jw-active-option):hover',
+                    '.jw-nextup-header'
+                ], 'color', activeColor);
+                addStyle([
+                    // menu active option
+                    '.jw-option.jw-active-option',
+                    // slider fill color
+                    '.jw-progress'
+                ], 'background', 'none ' + activeColor);
+            }
+
+            if (inactiveColor) {
+                // Apply inactive color
+                addStyle([
+                    // text color of many ui elements
+                    '.jw-text',
+                    // menu option text
+                    '.jw-option',
+                    // controlbar button colors
+                    '.jw-button-color',
+                    // toggle button
+                    '.jw-toggle.jw-off',
+                    '.jw-skip .jw-skip-icon',
+                    '.jw-nextup-body'
+                ], 'color', inactiveColor);
+                addStyle([
+                    // slider children
+                    '.jw-cue',
+                    '.jw-knob',
+                    '.jw-active-option',
+                    '.jw-nextup-header'
+                ], 'background', 'none ' + inactiveColor);
+            }
+
+            if (backgroundColor) {
+                // Apply background color
+                addStyle([
+                    // general background color
+                    '.jw-background-color'
+                ], 'background', 'none ' + backgroundColor);
+
+                addStyle([
+                    // for small player, set the control bar gradient to the config background color
+                    '.jw-flag-time-slider-above .jw-background-color.jw-controlbar'
+                ], 'background', 'none ' + backgroundColorGradient, true);
```

Comment:
If `_model.get('timeSliderAbove') === false` then we won't ever apply the `jw-flag-time-slider-above` right?  I think we can just let it be a dead style instead of adding to the size with the `if` statement if so.